stock_tracked_fifo_valuation/models/stock_move.py

```diff
@@ -8,10 +8,14 @@ class StockMove(models.Model):
     _inherit = "stock.move"
 
     def _action_done(self, cancel_backorder=False):
-        moves_tracked = self.filtered(lambda s: s.product_id.tracking != "none")
+        moves_tracked = self.filtered(
+            lambda s: s.product_id.tracking != "none" and not s._context.get("lots")
```

Comment:
This is quite inefficient. If lots is in context for 1 record, it is for all. Moreover if lots is set, there is only 1 record. Better to just use an if statement before.

I'll merge to keep things moving, but maybe tidy it up.